resources/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-Agent: *
+Disallow: /auth
+Disallow: /admin
```

Comment:
`Disallow: /helpdesk`, as well? After all, it's available for authenticated users only anyway.